modules/starfish/2.0/config/default.yaml

```diff
@@ -0,0 +1,51 @@
+lcr-modules:
+    
+    starfish:
+
+        dirs: 
+            
+            _parent: "__UPDATE__"
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            # Each key:value pair is the name of the variant caller and the path to its VCF file
+            # Each key should be unique in this set. 
+            vcf: 
+                strelka: "__UPDATE__"
+                
+
+            starfish_script: "{MODSDIR}/src/starfish.py"
+
+        scratch_subdirectories: []
+
+        options:
+            starfish_run: "--sample ALT --squash-ploidy --vennout-format pdf"
+            starfish_union: "-m both"
+
+        conda_envs:
+            bcftools: "{MODSDIR}/envs/bcftools-1.10.2.yaml"
+            starfish: "{MODSDIR}/envs/starfish.yaml"
```

Comment:
I'm not complaining, merely curious. What is our policy on when version numbers are and are not needed in these?

Reply:
I think they're usually named for the version of the core tool. Since the starfish one is just dependencies and Starfish itself is neither installed via Conda nor versioned it doesn't apply here.